src/CMakeLists.txt

```diff
@@ -203,7 +205,8 @@ target_link_libraries(keepassx_core
                       Qt5::Widgets
                       ${GCRYPT_LIBRARIES}
                       ${GPGERROR_LIBRARIES}
-                      ${ZLIB_LIBRARIES})
+                      ${ZLIB_LIBRARIES}
+                      ${OATH_LIBRARIES})
```

Comment:
Since this adds a new dependency, adding a -DWITH_XC_TOTP CMake option would be useful to optionally compile KeePassXC without TOTP support.